pkg/apis/camel/v1/integration_types.go

```diff
@@ -37,6 +37,7 @@ type IntegrationSpec struct {
 	Dependencies       []string                `json:"dependencies,omitempty"`
 	Profile            TraitProfile            `json:"profile,omitempty"`
 	Traits             map[string]TraitSpec    `json:"traits,omitempty"`
+	PodTemplate        *PodSpecTemplate        `json:"template,omitempty"`
```

Comment:
It may be more explicit to name the JSON field `podTemplate`?

Reply:
I kept the "template" to preserve the kubernetes naming: 
```
spec:
  template:
    spec: 
```

Reply:
Ah right, that's the whole point, I've been too prompt :)